olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -138,34 +138,34 @@ file(GLOB NETWORK_SOURCE
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/*.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/*.h"
 )
-set(NETWORK2_SOURCE
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkProxySettings.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkRequest.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkResponse.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkSettings.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/Network.cpp"
+
+set(EDGE_SDK_HTTP_SOURCES
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/http/NetworkProxySettings.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/http/NetworkRequest.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/http/NetworkResponse.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/http/NetworkSettings.cpp"
 )
 
 if (ANDROID)
 file(GLOB NETWORK_ANDROID_SOURCE
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/android/*.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/android/*.h"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network/socket/*.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/netwofrk/socket/*.cpp"
```

Comment:
oops

Reply:
:) New rule -> Never push code before you finish your first coffee.